### ISSUE TITLE
Do something about the unresponsive window during the archive scan

### ISSUE THREAD

**Issue by lhog** [beyond-all-reason/spring#1654]
This is when the game is started.

**Comment by WatchTheFort** [comment]
Will this be mitigated with the new lobby? Since it won't be running an instance of the engine, the engine can be started in an invisible window and do its archive scan in the background?

**Comment by lhog** [comment]
The issue is inherent to the engine, no matter what lobby is in use

**Comment by 12345swordy** [comment]
Yes please, it makes it hard for me to debug the engine while running.

**Comment by lhog** [comment]
> Yes please, it makes it hard for me to debug the engine while running.

Unsure what you're talking about. The scan happens only once per branch name per asset. You shouldn't experience that every time.
Please, place the break-point while you experience that issue and see where the engine spends its time.

**Comment by 12345swordy** [comment]
> Unsure what you're talking about. 
I can't not switch to visual studio IDE during the archive scan.

**Comment by lhog** [comment]
> > Unsure what you're talking about.
> 
> I can't not switch to visual studio IDE during the archive scan.

Usually you can if you don't run in the full screen mode. Otherwise you probably experience some different issue.
